symbology/symbol-dictionary/README.md

```diff
@@ -1,28 +1,26 @@
-# Symbol Dictionary
+# Search symbol dictionary
 
 Find symbols within the mil2525d specification that match a keyword.
 
-![](SymbolDictionary.png)
+![Image of search symbol dictionary](SymbolDictionary.png)
 
 ## Use case
 
 You can use support for military symbology to allow users to report changes in the field using the correct military symbols.
 
 ## How to use the sample
 
-By default, leaving the fields blank and searching will return all symbols.
+By default, leaving the fields blank and hitting search will find all symbols.
 
-To filter symbols, enter text into one or multiple seach boxes and click the Search for Symbols button.
-
-Click the Clear button to clear previous search results.
+To search for certain symbols, enter text into one or more search boxes and click 'Search for symbols'. Results are shown in a list. Pressing 'Clear' will reset the search.
 
 ## How it works
 
-1. Create a symbol dictionary with the mil2525d specification by passing the string "mil2525d" to the `SymbolDictionary` constructor.
+1. Create a symbol dictionary with the mil2525d specification by passing the path to a .stylx file to the `SymbolDictionary.createFromFile(path)` constructor.
 2. Create `StyleSymbolSearchParameters`.
-3. Add members to the names, tags, symbolClasses, categories, and keys list fields of the search parameters.
+3. Add members to the `names`, `tags`, `symbolClasses`, `categories`, and `keys` list fields of the search parameters.
```

Comment:
removing these backticks since they aren't API refs
```suggestion
3. Add members to the names, tags, symbolClasses, categories, and keys list fields of the search parameters.
```